subsys/bluetooth/host/smp.c

```diff
@@ -3686,6 +3692,13 @@ static u8_t smp_security_request(struct bt_smp *smp, struct net_buf *buf)
 		auth = req->auth_req & BT_SMP_AUTH_MASK;
 	}
 
+	if (IS_ENABLED(CONFIG_BT_BONDING_REQUIRED) &&
+		(!bondable ||
```

Comment:
Indentation should be 1 tab + 4 spaces here, now it appears to be 2 tabs. Also I think it is better without the linebreak after ||.
Also I find it easier to read it when when you put the `!` outside the expression, that matches the comment better, i.e "not both" matches !(a && B) better (but that is a matter of taste I guess). 
```
	if (IS_ENABLED(CONFIG_BT_BONDING_REQUIRED) &&
	    !(bondable && (auth & BT_SMP_AUTH_BONDING))) {
```

Reply:
Thanks for the swift feedback. Need to get used to this indentation scheme :) I'll have a fix for this in a minute.